mimic/test/test_nova.py

```diff
@@ -4,6 +4,7 @@
 from __future__ import absolute_import, division, unicode_literals
 
 import json
+import random
```

Comment:
@tawalton - yes, perhaps creating one test case for each server image programatically.